assets/js/components/DashboardMainApp.js

```diff
@@ -230,6 +232,14 @@ export default function DashboardMainApp() {
 				</Fragment>
 			) }
 
+			{ ! viewOnlyDashboard && (
+				<Fragment>
```

Comment:
This should be combined with the same expression above as it's unnecessarily duplicated here.